src/elm/View/MainText.elm

```diff
@@ -1,6 +1,6 @@
 module View.MainText exposing (viewBottom, viewTop)
 
-import Data
+import Data exposing (SectionId(..))
```

Comment:
Tiny observation.

I've been using fully qualified types inline eg. `Data.Section1` apart from Copy Keys but it doesn't matter.